paderbox/io/hdf5.py

```diff
@@ -8,6 +8,51 @@
 
 __all__ = ['dump_hdf5', 'update_hdf5', 'load_hdf5']
 
+def join_paths(path, name):
```

Comment:
Why not using something like `re.sub('/+', '/', name if name.startswith('/') else f'{path}/{name}')`?
That mirrors `os.path.join` for linux.